airbyte-integrations/connectors/source-twilio-taskrouter/source_twilio_taskrouter/twilio_taskrouter.yaml

```diff
@@ -0,0 +1,76 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+  requester:
+    url_base: "https://taskrouter.twilio.com"
+    http_method: "GET"
+    authenticator:
+      type: BasicHttpAuthenticator
+      username: "{{ config['account_sid'] }}"
+      password: "{{ config['auth_token'] }}"
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
+    requester:
+      $ref: "*ref(definitions.requester)"
+  base_stream:
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+  allworkspaces_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "allworkspaces"
+      path: "/v1/Workspaces"
+      primary_key: "sid"
+    retriever:
+      $ref: "*ref(definitions.base_stream.retriever)"
+      record_selector:
+        extractor:
+          field_pointer: ["workspaces"]
+  workspaces_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "workspaces"
+      primary_key: "sid"
+      path: "/v1/Workspaces/{{config['sid']}}"
+  workers_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "workers"
+      primary_key: "sid"
+      path: "/v1/Workspaces/{{config['sid']}}/Workers"
+    retriever:
+      $ref: "*ref(definitions.base_stream.retriever)"
+      record_selector:
+        extractor:
+          field_pointer: ["workers"]
+      paginator:
+        type: "DefaultPaginator"
+        page_size_option:
+          inject_into: "request_parameter"
+          field_name: "page_size"
+        pagination_strategy:
+          type: "PageIncrement"
+          page_size: 2
+        page_token_option:
+          inject_into: "request_parameter"
+          field_name: "page"
+        url_base: "https://taskrouter.twilio.com"
+   
+streams:
+  - "*ref(definitions.workspaces_stream)"
+  - "*ref(definitions.allworkspaces_stream)"
+  - "*ref(definitions.workers_stream)"
+
+
+check:
+  stream_names:
+    - "allworkspaces"
```

Comment:
I think you only need one stream to verify the check

Reply:
sure I'll remove the other two streams

Reply:
it still requires fetching all 3 streams to verify the connectivity. Can you simplify it? Thanks.

Reply:
Yeah i seemed to have made the new changes in only my topic branch.  I'll push the changes to this branch